packages/webawesome/docs/assets/styles/utils.css

```diff
@@ -79,6 +79,20 @@
       font-size: var(--wa-font-size-xs);
       --max-width: unset;
     }
+
+    &.shrink {
+      font-size: round(calc(var(--wa-font-size-2xs) / 1.125), 1px); /* ~10px */
+    }
+  }
+
+  /* planned badge */
+  wa-badge.planned {
+    background-color: var(--wa-color-neutral-fill-quiet);
+
+    + wa-tooltip {
```

Comment:
We may want to think about making a more general rule for tooltip `font-size` on badges, but I thought that blanket call was a bit premature with this being the second instance. Letting it ride like we're in WAs Vegas!